src/view/view.manager.ts

```diff
@@ -60,4 +60,10 @@ export class ViewManager {
 
         return channelViewMap;
     }
+
+    static async getChannelViews(channelId: string) {
+        const [channelViews] = await VideoRepository.getChannelsViews([channelId]);
+
+        return channelViews.views;
```

Comment:
If channel not exists there will be an error "Cannot read property 'views' of undefined"
```return channelViews ? channelViews.views : 0```

Reply:
OK. Done.